runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/Environments.java

```diff
@@ -448,6 +451,39 @@ public static String getExternalServiceAddress(PortablePipelineOptions options)
     return environmentOption;
   }
 
+  /**
+   * Configure log manager's default log level and log level overrides from the sdk harness options,
+   * and return the list of configured loggers.
+   */
+  public static List<java.util.logging.Logger> getConfiguredLoggerFromOptions(
```

Comment:
Was trying to find the best place to put these piece of code that sets log levels. It is now used by both `:sdks:java:harness` and `:runners:flink`. Seems like this module is a good place as there are other static public methods related to environments here.

Reply:
is `SdkHarnessOptions.java` accessible from `:runners:flink` ?

Reply:
I see you do import that file in FlinkPipelineRunner.java so it must be

Reply:
have you considered it as a home for this helper?

Reply:
SdkHarnessOptions.java is in sdks:java:core and yes it is accessible